src/api/app/models/bs_request_action/differ.rb

```diff
@@ -1,3 +1,5 @@
+require 'bs_request_action/differ/for_source'
```

Comment:
Why?

Reply:
Try running ./spec/controllers/webui/request_controller_spec.rb:128. You should get an exception (see commit message).

Reply:
But I'm wondering why it worked before?

Reply:
I got this error when disabling VCR. Like I wrote;-)

Reply:
Fine for me